recipes/default.rb

```diff
@@ -8,3 +8,4 @@
 include_recipe 'curl'
 
 elasticsearch_user 'elasticsearch'
+elasticsearch_install 'elasticsearch'
```

Comment:
The string is actually just a resource name, not something particular to the LWRP DSL. It could be anything; every Chef resource requires some sort of string name, though.

There's actually some examples embedded in the `test/fixtures/cookbooks` directory, but I agree, we should document all of the usage and options. I will work on getting that in the readme.